credit_card.rb

```diff
@@ -11,4 +11,8 @@ def initialize
   def charge(amount)
     @balance + amount
   end
+
+  def payment(amount)
+    @balance + amount
```

Comment:
This method should subtract the amount.  i.e. I paid $25.00 on my credit card, so my balance should go down by $25.00.